docs/APIRef.ActionsOnElement.md

```diff
@@ -242,12 +242,16 @@ jestExpect(multipleMatchedElements.elements.length).toBe(5);
 jestExpect(multipleMatchedElements.elements[0].identifier).toBe('FirstElement');
 ```
 
-### `takeScreenshot()` Android Only
+### `takeScreenshot(name)` Android Only
```

Comment:
I'd suggest using `takeScreenshot([name])` square brackets convention for marking optional args.

Reply:
np